elife-1234567890-v2/elife-1234567890-v2.xml

```diff
@@ -21,6 +21,8 @@
             <article-id pub-id-type="doi">10.7554/eLife.1234567890</article-id>
             <!-- Specific version DOI -->
             <article-id pub-id-type="doi" specific-use="version">10.7554/eLife.1234567890.4</article-id>
+            <!-- Article version used to distinguish reviewed preprints from versions of record. For RPs, value would be "reviewed preprint" -->
+            <article-version article-version-type="publication-state">version of record</article-version>
```

Comment:
I think we can only have one article-version element. The validator does not like it if I try to add a second....

Reply:
I think if we used more than one, they would have to be contained in [`<article-version-alternatives>`](https://jats.nlm.nih.gov/archiving/tag-library/1.3/element/article-version-alternatives.html) in order to be valid JATS.

Reply:
Of course! Sorry, should have seen that; just assumed it worked like article-id for some reason.

Reply:
After some thought, I've opted for RP-1, RP-2, RP-3 and VOR-1, VOR-2, VOR-3 etc. I don't think there is much utility here for a 1.0, 1.1 etc style of version number, since it would make accounting for old model content a bit more of a pain, and it's probably more useful to clearly indicate in this term what kind of content the article is (RP or VoR).

Does that sound sensible?

Reply:
We have decided to proceed without this second article-version element.